compiler/parser/src/parser/file.rs

```diff
@@ -50,15 +51,12 @@ impl ParserContext<'_> {
                     let (id, function) = self.parse_function()?;
                     functions.insert(id, function);
                 }
-                Token::Const if self.peek_is_function() => {
-                    let (id, function) = self.parse_function()?;
-                    functions.insert(id, function);
-                }
                 Token::Identifier(sym::test) => return Err(ParserError::test_function(self.token.span).into()),
-                Token::Function => {
+                Token::Function | Token::Transition => {
```

Comment:
Should this include the `inline` token, or is that a future feature? (I saw that `CallType` includes an `Inline` entry.)